tests/integration/test_hedged_requests/test.py

```diff
@@ -88,7 +90,29 @@ def check_changing_replica_events(expected_count):
     assert int(result) == expected_count
 
 
+def update_configs(node_1_sleep_in_send_tables_status=0, node_1_sleep_in_send_data=0,
+                    node_2_sleep_in_send_tables_status=0, node_2_sleep_in_send_data=0,
+                    node_3_sleep_in_send_tables_status=0, node_3_sleep_in_send_data=0):
+    NODES['node_1'].replace_config(
+        '/etc/clickhouse-server/users.d/users1.xml',
+        config.format(sleep_in_send_tables_status_ms=node_1_sleep_in_send_tables_status, sleep_in_send_data_ms=node_1_sleep_in_send_data))
+
+    NODES['node_2'].replace_config(
+        '/etc/clickhouse-server/users.d/users1.xml',
+        config.format(sleep_in_send_tables_status_ms=node_2_sleep_in_send_tables_status, sleep_in_send_data_ms=node_2_sleep_in_send_data))
+
+    NODES['node_3'].replace_config(
+        '/etc/clickhouse-server/users.d/users1.xml',
+        config.format(sleep_in_send_tables_status_ms=node_3_sleep_in_send_tables_status, sleep_in_send_data_ms=node_3_sleep_in_send_data))
+
```

Comment:
Do we need `SYSTEM RELOAD CONFIG` here?

Reply:
But we update configs automatically if they were changed. After these replaces we have check_settings that ensure that configs were reloaded and we have actual settings.